standards/request-response.md

```diff
@@ -666,7 +666,7 @@ For each HTTP method with only the status codes specified below, API developers
 
 | Status Code               | GET | POST | PUT | PATCH | DELETE | HEAD | OPTIONS |
 |---------------------------|-----|------|-----|-------|--------|------|---------|
-| 200 OK                    |  X  |  X   |     |       |        |  X   |   X     |
+| 200 OK                    |  X  |  X   |     |   X   |        |  X   |   X     |
```

Comment:
I assume that PUT and PATCH would be at parity for the most part regarding the status codes they can return @travisgosselin?